orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>I think item 5 is not correctly formatted or connected to the below section on Geostationary orbits. Can you double check why that is  and fix that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I have added the brackets from the title to it, it should work now:</p><p>5. [](content:geostationary-orbits-(GEO))</p>